tests/test_cli.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+from cli.main import make_job_create_request
+
+
+class TestCLI(unittest.TestCase):
+
+    def test_create_job(self):
+        self.maxDiff = None
```

Comment:
What is the effect of this assignment?

Reply:
Otherwise unittest fails because the objects to compare are too big, I will add a comment in the code.

Reply:
done